src/js/views/patients/actions/actions_views.js

```diff
@@ -254,6 +254,82 @@ const DueComponent = Component.extend({
   },
 });
 
+// Every 15 mins for 24 hours starting at 7am
+const start = moment({ hour: 6, minute: 45, second: 0 });
+
+const times = _.times(96, function() {
+  return { time: start.add(15, 'minutes').format('LT') };
+});
+
+times.unshift({ time: null });
+
+const DueTimeComponent = Droplist.extend({
+  isCompact: false,
+  isSelectlist: true,
+  getClassName(time) {
+    const isCompact = this.getOption('isCompact');
+
+    if (time === null && isCompact) {
+      return 'button-secondary--compact actions__time is-icon-only';
+    }
+    if (isCompact) {
+      return 'button-secondary--compact actions__time ';
+    }
+
+    return 'button-secondary actions__time w-100';
+  },
+  getTemplate(time, isCompact) {
+    if (!time && this.getOption('isCompact')) {
+      return hbs`{{far "clock"}}`;
+    }
+
+    if (!time) {
+      return hbs`{{far "clock"}}{{@intl.patients.actions.actionsViews.dueTimeComponent.placeholderText}}`;
+    }
+
+    return hbs`{{far "clock"}} {{time}}`;
+  },
+  viewOptions() {
+    const time = (this.getState('selected')) ? this.getState('selected').get('time') : null; // not great bob
```

Comment:
done because if `'selected'` is null everything blows up

Reply:
when/why is selected null?

Reply:
well apparently never...don't know why I had to do this previously, but don't need to anymore...